python/opendht.pyx

```diff
@@ -57,6 +57,20 @@ cdef inline void shutdown_callback(void* user_data) with gil:
         cbs['shutdown']()
     ref.Py_DECREF(cbs)
 
+cdef inline bool get_filter(const cpp.Value& value, void *user_data) with gil:
+    cbs = <object>user_data
+    if 'filter' in cbs and cbs['filter']:
+        # Deep of const Value to Python level Value
```

Comment:
The filter feature in the Python bindings could also be implemented by calling the python filter from the get callback wrapper (this file, in the get_callback wrapper, just bellow)